.github/std-unexploded.yml

```diff
@@ -0,0 +1,107 @@
+# yq '. | explode(.)' std-unexploded.yml > workflows/std.yml
+name: CI/CD
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+env:
+  AWS_REGION: us-east-1
+  AWS_ROLE_ARN: arn:aws:iam::926093910549:role/lace-ci
+  # NIX_UPLOAD_CACHE: s3://lace-nix-cache?region=us-east-1
+  DISCOVERY_USER_NAME: gha-runner
+  DISCOVERY_KNOWN_HOSTS_ENTRY: "65.109.126.156 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEOVVDZydvD+diYa6A3EtA3WGw5NfN0wv7ckQxa/fX1O"
+
+permissions:
+  id-token: write
+  contents: read
+
+concurrency:
+  group: std-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  discover:
+    outputs:
+      hits: ${{ steps.discovery.outputs.hits }}
+    runs-on: [self-hosted, discovery]
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@main
+        with:
+          role-to-assume: ${{ env.AWS_ROLE_ARN }}
+          aws-region: ${{ env.AWS_REGION }}
+          # account is part of ecr url, thus part of `hits` output and needs to pass
+          mask-aws-account-id: false
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      # TODO: uncomment when nixbuild works well together with
+      #       nix daemon mode on hosted runners
+      # - uses: nixbuild/nixbuild-action@v17
+      #   with:
+      #     nixbuild_ssh_key: ${{ secrets.SSH_PRIVATE_KEY }}
+      #     generate_summary_for: job
```

Comment:
Why do we need nixbuild.net on discovery? Discovery is not doing much and CI runs are likely to be faster without nixbuild.net since it introduces network overhead

Reply:
This decisively depends on the amount of Import From Derivation. My recollectoion is that we still haven't reduced the amount of IFD in this repository which is why the goal to mitigate it with remote builds remains on the agenda as soon as we have compatibility.

Reply:
As far as i am aware we are doing exactly zero IFD in this project

Reply:
Lets remove this comment and we can resolve this conversation

Reply:
It's a bit linked to leaving https://github.com/input-output-hk/cardano-js-sdk/pull/812#discussion_r1268714723. I intent to follow up on this, later and as discussed today in our standup, hence the TODO.